File: PHP/2-OCP/OCP-Violacao/DebitoConta.php
<?php

class DebitoConta
{
    const CORRENTE = 1;
    const POUPANCA = 2;
    function debitar(float $valor, string $conta, int $tipo)
    {
        if ($tipo == self::CORRENTE) {
            echo 'corrente';
            // Debita Conta Corrente
        }
        if ($tipo == self::POUPANCA) {
            echo 'poupança';
            // Valida Aniversário da Conta
            // Debita Conta Poupança
        }
    }
}

File: Python/2-OCP/OCP-Violacao/DebitoConta.py
class DebitoConta:
    def __init__(self):
        self.Corrente = 1
        self.Poupanca = 2
        pass
    
    def debitar(self, valor: int, conta: str, tipo: int):
        if tipo == self.Corrente:
            print('Corrente')
        
        if tipo == self.Poupanca:
            print('Poupanca')
            # Valida Aniversário da Conta
            # Debita Conta Poupança